index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Catpplication</title>
+    <link rel="stylesheet" href="./src/main.css">
+</head>
+<body>
+    <header>
+        <h1>
+            Please adopt one of us!
+        </h1>
+    </header>
+    
+    <main>
+        <section class="intro">
+            <p>
+                <img class="typing-cat" src="./assets/cat.gif" alt="A cat in a tie and glasses typing at a keyboard">
+            <br> We ain't kitten around. Bring one of us home meow! <br> Fill out the simple form and paws to congradulate yourself!
+            <nav>
```

Comment:
This seems awkward to me. A nav usually lives in the header or footer with links for navigation. For your purpose you just have a anchor link that takes you to another page in your section. I would consider losing the ```p``` and ```nav``` tags and replace it with a ```div``` if you need a parent container or just style the ```a```tag.